vscode TS language server fails to detect references

fresh clone of master

Additional context: 

```
npx tsc -b                              
packages/web/src/add.ts:1:31 - error TS2307: Cannot find module '@ahoopen/utils' or its corresponding type declarations.

1 import { multiplyByTwo } from "@ahoopen/utils";
                                ~~~~~~~~~~~~~~~~


Found 1 error.
```

--- a/packages/utils/package.json
+++ b/packages/utils/package.json
@@ -1,25 +1,25 @@
 {
-    "name": "@ahoopen/utils",
-    "version": "1.0.0",
-    "main": "index.js",
-    "types": "index.d.ts",
-    "license": "MIT",
-    "repository": {
-        "type": "git",
-        "directory": "packages/web",
-        "url": "https://github.com/ahoopen/typescript-mono-repo.git"
-    },
-    "scripts": {
-        "clean": "rimraf dist && rimraf tsconfig.tsbuildinfo",
-        "prepack": "yarn build",
-        "build": "yarn clean && tsc --build && cp \"./package.json\" ./dist/",
-        "test": "echo 'no test'",
-        "lint": "eslint \"./src/**/*.{ts,tsx}\" --max-warnings=0"
-    },
-    "dependencies": {
-        "react": "16.13.1"
-    },
-    "devDependencies": {
-        "@types/react": "16.9.48"
-    }
+  "name": "@ahoopen/utils",
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "directory": "packages/web",
+    "url": "https://github.com/ahoopen/typescript-mono-repo.git"
+  },
+  "scripts": {
+    "clean": "rimraf dist && rimraf tsconfig.tsbuildinfo",
+    "prepack": "yarn build",
+    "build": "yarn clean && tsc --build && cp \"./package.json\" ./dist/",
+    "test": "echo 'no test'",
+    "lint": "eslint \"./src/**/*.{ts,tsx}\" --max-warnings=0"
+  },
+  "dependencies": {
+    "react": "16.13.1"
+  },
+  "devDependencies": {
+    "@types/react": "16.9.48"
+  }
 }

--- a/packages/web/package.json
+++ b/packages/web/package.json
@@ -1,8 +1,8 @@
 {
   "name": "@ahoopen/web",
   "version": "1.0.0",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "license": "MIT",
   "repository": {
     "type": "git",

